AppCenter/AppCenterTests/MSUserDefaultsTests.m

```diff
@@ -4,9 +4,9 @@
 #import "MSAppCenterUserDefaults.h"
 #import "MSAppCenterUserDefaultsPrivate.h"
 #import "MSLoggerInternal.h"
-#import "MSTestFrameworks.h"
 #import "MSUtility.h"
 #import "MSWrapperLogger.h"
+#import <XCTest/XCTest.h>
```

Comment:
Why do we need change imports here? This one was already imported in `MSTestFrameworks.h`

Reply:
There was a problem with `XCTAssertNoThrow`. It didn't until I added `#import <XCTest/XCTest.h>`.

Reply:
XCTAssertNoThrow won't work without it

Reply:
@MatkovIvan Reverted, seems working fine without it indeed. Please take a look.